docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -3,41 +3,70 @@ Datasets
 Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
 for preparing datasets for training and validating VAD and speaker embedding models respectively.
 
-For Speaker Diarization inference, ``diarizer`` expects either list of paths to audio files or a file containing absolute paths to audio files. 
+For Speaker Diarization inference, ``diarizer`` expects following arguments, which needs to be created by user depending on dataset:
 
-For generating a file that contains paths to audio files (which we call as ``scp file``), you can simply use ``find`` bash command as shown below:
+  - paths2audio_files
+  - oracle_num_speakers
+  - path2groundtruth_rttm_files [Optional]
+  - oracle_vad_manifest (if vad/rttm files are known)[Optional but suggested for good performace. This avoids FPR and TPR]
 
+paths2audio_files:
+  - a file containing absolute paths to audio files or list of paths to audio files. 
+  For generating a file that contains paths to audio files (which we call as ``scp file``), you can simply use ``find`` bash command as shown below:
 .. code-block:: bash
 
   find $PWD/{data_dir} -iname '*.wav' > path_to_audiofiles.scp
 
+.. note::
+  We expect audio and the corresponding RTTM to have the same base name and the name should be unique (uniq-id).
```

Comment:
We expect audio and the corresponding RTTM -> We expect audio and the corresponding RTTM files